docs/licensing/chibios.md

```diff
@@ -3,42 +3,42 @@
-We've been in touch with the ChibiOS sales team to clarify this as much as possible.
+This document provides a general overview of the licensing options available when using [ChibiOS](http://chibios.org) as the **nanoFramework** RTOS component.
+The nanoFramework team have been in contact with the ChibiOS sales team to ensure accuracy of the following information at time of print (October 2017).
 
-For details and to discuss your particular situation we strongly recommend getting in touch with the ChibiOS sales [here](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
+For details or to discuss your particular situation, we strongly recommend getting in touch with the ChibiOS [sales team](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
 
 
-## What exactly is being used from ChibiOS in **nanoFramework**?
+## What exactly is being used from ChibiOS in **nanoFramework firmware**?
 
-**nanoFramework** is built against _unmodified_ ChibiOS sources. It's using:
-* The [HAL](http://chibios.org/dokuwiki/doku.php?id=chibios:product:hal:start), which is released under Apache License 2.0 meaning that it's 100% free also for commercial use.
-* The [RT](http://chibios.org/dokuwiki/doku.php?id=chibios:product:rt:start), which is released under GPL3. See bellow about the licensing options according to your use. 
+**nanoFramework** is built against _unmodified_ ChibiOS sources via a mirror. It's using:
+* The [HAL](http://chibios.org/dokuwiki/doku.php?id=chibios:product:hal:start), which is released under Apache License 2.0 meaning that it's 100% free to use or distribute without royalties for any purpose.
+* The [RT](http://chibios.org/dokuwiki/doku.php?id=chibios:product:rt:start), is released under GPL3. See below for the licensing options which maybe different depending on your particular use. 
 
 
 ## Can I use ChibiOS freely for my hobby or personal development at home?
 
-Yes, using ChibiOS in FOSS project is perfectly fine.
+Yes, using ChibiOS in a Free and Open Source Software project or for personal use is perfectly fine.
 
 
 ## Can I use ChibiOS freely if I'm developing a commercial product?
 
-Here there are several options:
-* Yes, if you are OK with [GPL3](https://www.gnu.org/licenses/gpl.html) licensing terms. Basically you have to open your source code.
-* Yes, if you are OK with ChibiOS publicizing your use of it _and_ you have to clearly mention that your product is using ChibiOS.
+Yes but please be aware of the following options:
+* if you modify the [firmware](nanoframework/nf-interpreter) in any way and you are okay with [GPL3](https://www.gnu.org/licenses/gpl.html) licensing terms. Basically the firmware **MUST** keep your firmware open source.
+* if you modify the [firmware](nanoframework/nf-interpreter) in any way and are okay with ChibiOS publicizing your use of it _and_ you clearly mention that your product is using ChibiOS.
```

Comment:
same as above. changing nF code is OK, so this part is not correct

Reply:
Same reasoning as above. The proposed changes shouldn't be there because that is not correct.